tests/assemblyscript/package.json

```diff
@@ -7,7 +7,9 @@
     "test": "echo \"Error: no test specified\" && exit 1",
     "asc": "asc",
     "lint": "eslint . --ext .ts",
-    "build": "ts-node buildscripts/build.ts"
+    "build": "ts-node buildscripts/build.ts",
+    "prettier-format-check": "prettier '**/*.ts' --check",
```

Comment:
should we also update CI build scripts to enforce the format?

Reply:
fixed!

Reply:
added!